lib/chat_api_web/channels/conversation_channel.ex

```diff
@@ -161,4 +190,8 @@ defmodule ChatApiWeb.ConversationChannel do
       _ -> false
     end
   end
+
+  defp user_id_of_signature(_signature) do
+    1
```

Comment:
it should receive the message, signature and validate it

if valid, return an user_id so that the message looks like it's from an agent?